devices/dawon_dns.js

```diff
@@ -272,4 +273,37 @@ module.exports = [
         },
         exposes: [e.battery(), e.switch()],
     },
+    {
+        zigbeeModel: ['KB-HD100-ZB'],
+        model: 'KB-HD100-ZB',
+        vendor: 'Dawon DNS',
+        description: 'IOT Card holder',
+        fromZigbee: [fz.dawon_card_holder],
+        toZigbee: [],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['ssIasZone']);
+            const payload = [{
+                attribute: 'zoneState', minimumReportInterval: 0, maximumReportInterval: 3600, reportableChange: 0}];
+            await endpoint.configureReporting('ssIasZone', payload);
+        },
+        exposes: [exposes.binary('card', ea.STATE, true, false)
```

Comment:
Based on `fz. dawon_card_holder`, `e.battery_low()` should also be here.

Reply:
I fix it.